app/server/login.py

```diff
@@ -0,0 +1,35 @@
+import json
+
+import requests as r
+from fastapi import APIRouter
+from starlette.responses import RedirectResponse
+
+from app.config.config import settings
+
+router = APIRouter()
+
+
+@router.get("/login")
+def authorize():
+    res = r.get(
+        url="https://github.com/login/oauth/authorize",
+        data={
+            "client_id": settings.CLIENT_ID,
+            "redirect_uri": "http://localhost:8080/token",
+        },
+    )
+    return RedirectResponse(url=res.url)
+
+
```

Comment:
This will be handled from the frontend..

Reply:
We'd have to return a code

Reply:
Got it.